README.md

```diff
@@ -1,2 +1,76 @@
-# intergalactic-most-wanted-list
-A database for malware checksums for use with the WallE malware cleanup script
+# Intergalactic Most Wanted List
+A database for malware checksums for use with the [WallE](https://github.com/usegalaxy-eu/WallE) malware cleanup script.
+
+This database in form of a yaml file stores malware checksums and its metadata
+schema:
+~~~yaml
+class:
+  program:
+    version:
+      severity: [high, medium, low]
+      description: "optional info"
+      checksums:
+        crc32: <checksum crc32, gzip algorithm, integer representation>
+        sha1: <checksum sha1, hex representation>
+~~~
+Contributions are always welcome. Here is a more detailed explanation how the schema works.
+### Class
+What purpose does it serve, e.g. mining, (port-)scanner, multi-purpose...  
+
+**Not** a `class` would be `nmap`, it is a specific tool for scanning and would be a `program` under the `scanning` class.
```

Comment:
```suggestion
`nmap` would **not** be a `class`, because it is a specific tool for scanning and would be a `program` under the `scanning` class.
```